rpcd/playbooks/roles/rpc_pre_upgrade/tasks/main.yml

```diff
@@ -25,6 +25,8 @@
 
 # Gather swift-recon md5 output
 - include: gather_swift_stats.yml
+  when: >
+    groups['swift_proxy']|length > 0
```

Comment:
According to @andymcc , we define the swift_proxy group by default, so checking the length is enough.